### PR TITLE
[OoT] addAtStartPosIndex -> addStartPosAtIndex

### PR DESCRIPTION
**Pull request by Yanis42**
From what I understand this is a typo (the parameters match addStartPosAtIndex), randomly found this while looking at the OoT exporter

### CODE REVIEW

**Comment by Zelllll** [comment]
I think this might have been related to a bug that existed a long time ago where start positions would be exported incorrectly, what does that function do exactly?

**Comment by Yanis42** [comment]
I don't really know what this is for, I randomly saw that typo while checking the purpose of  ``actorProp`` in ``addStartPosition`` (I'm working on the exporter for the detailed parameter panel thing I'm doing)

Apparently it fills a dictionnary of start positions, I'm not sure why though, I haven't focused a lot of what Fast64 actually does outside drawing things

**Comment by Zelllll** [comment]
Yeah that makes sense. I'm kind of confused why it appeared to work with the typo existing though.

**Review by Zelllll — APPROVED** [review]

**Comment by Yanis42** [comment]
I guess no one ever tried to export a scene with the scene setup preset set to custom, but yeah I'm really surprised Blender never complained about that

**Comment by Zelllll** [comment]
Should we close this since you added it to your other PR?

**Comment by Yanis42** [comment]
I'll close this yeah, it's not an important change anyway so this can wait until the #56 is merged

**Pull request closed without merging**
